CHANGELOG.md

```diff
@@ -25,6 +25,29 @@
   });
   ```
 
+- Adds feedback form ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))
+
+  You can add the form component in your UI and customise it like:
+  ```jsx
+  import { FeedbackFormScreen } from "@sentry/react-native";
+  ...
+  <FeedbackFormScreen
+    {...props}
+    closeScreen={props.navigation.goBack}
+    styles={{
+      submitButton: {
+        backgroundColor: '#6a1b9a',
+        paddingVertical: 15,
+        borderRadius: 5,
+        alignItems: 'center',
+        marginBottom: 10,
+      },
+    }}
+    text={{namePlaceholder: 'Fullname'}}
+  />
+  ```
+  Check [the documentation](https://docs.sentry.io/platforms/react-native/user-feedback/) for more configuration options.
```

Comment:
The documentation will be updated with https://github.com/getsentry/sentry-docs/pull/11980 (WIP)